plugin/storage/cassandra/spanstore/reader.go

```diff
@@ -377,7 +377,7 @@ func (s *SpanReader) queryByDuration(ctx context.Context, traceQuery *spanstore.
 
 func (s *SpanReader) queryByServiceNameAndOperation(ctx context.Context, tq *spanstore.TraceQueryParameters) (dbmodel.UniqueTraceIDs, error) {
 	//lint:ignore SA4006 failing to re-assign context is worse than unused variable
```

Comment:
in this case the comment should be removed

Reply:
updated, PTAL